skxray/roi.py

```diff
@@ -59,38 +60,35 @@
 logger = logging.getLogger(__name__)
 
 
-def rectangles(num_rois, roi_data, image_shape):
+def rectangles(coords, shape):
     """
     Parameters
     ----------
-    num_rois: int
-        number of region of interests(roi)
+    coords: ndarray
+        coordinates of the upper-left corner and width and height of each
+        rectangle: e.g., [(x, y, w, h), (x, y, w, h)]
 
-    roi_data: ndarray
-        upper left co-ordinates of roi's and the, length and width of roi's
-        from those co-ordinates
-        shape is [num_rois][4]
-
-    image_shape : tuple
-        2 element tuple defining the number of pixels in the detector.
-        Order is (num_rows, num_columns)
+    shape : rr, cc : tuple
+        Image shape which is used to determine the maximum extent of output
+        pixel coordinates.
 
     Returns
     -------
-    labels_grid : array
-        indices of the required rings
-        shape is ([image_shape[0]*image_shape[1]], )
+    label_array : rr, cc: array
```

Comment:
See scikit-image. This is what they do. Are you sure you don't want to
follow that?
On Fri, Apr 24, 2015 at 8:45 AM Thomas A Caswell <notifications@github.com>
wrote:

> In skxray/roi.py
> <https://github.com/Nikea/scikit-xray/pull/272#discussion_r29045048>:
>
> >
> >      Returns
> >      -------
> > -    labels_grid : array
> > -        indices of the required rings
> > -        shape is ([image_shape[0]*image_shape[1]], )
> > +    label_array : rr, cc: array
>
> ditto re doc format
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Nikea/scikit-xray/pull/272/files#r29045048>.
>